eth/db/account.py

```diff
@@ -435,6 +460,27 @@ def persist(self) -> None:
             self._batchdb.commit_to(write_batch, apply_deletes=False)
         self._root_hash_at_last_persist = new_root_hash
 
+        return witness_index
+
+    def _get_accessed_node_hashes(self) -> Set[Hash32]:
+        return cast(Set[Hash32], self._raw_store_db.keys_read)
+
+    @to_dict
+    def _get_witness_metadata(self) -> Iterable[Tuple[Address, AccountQueryTracker]]:
```

Comment:
I don't know if it's possible to call `get_storage` without first reading something from the `get_account`, but if it is possible then we'll miss some reads here. Here, we only read out of `_account_stores` for all the addresses in `_accessed_accounts`, but I think nothing `get_storage` calls ever adds to `_accessed_accounts`.

Reply:
> I think nothing `get_storage` calls ever adds to `_accessed_accounts`.

Hm, if the address store is in cache, then it won't trigger, but I don't know how it could get in cache without calling `_get_address_store()` first. That one does add the address to the list of accessed accounts, via:
`get_storage()` -> `_get_address_store()` -> `_get_storage_root()` -> `_get_account()` -> `_get_encoded_account()` -> adds to `_accessed_accounts`